cfgov/diversity_inclusion/jinja2/diversity_inclusion/privacy.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'layout-2-1-bleedbar.html' %}
+
+{% block meta_viewport %}
+    {{ super() }}
+    <meta name="robots" content="noindex">
+{% endblock %}
+
+{% block title -%}
+    Privacy Act Statement
+{%- endblock %}
+
+{% block desc -%}
+    TODO: Come back to this
```

Comment:
Should these TODOs in the templates still be here?

Reply:
Good catch! Updated in a2b6197.